reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -286,6 +287,98 @@ public EmitResult getReason() {
 		ManySpec many();
 	}
 
+	public interface RootUnsafeSpec extends RootSpec {
+
+		/**
+		 * Connect a {@link Sinks.Many} to an arbitrary {@link Publisher} of compatible type.
+		 * This is considered an unsafe operation because connecting implies adhering to some rules
+		 * and ceremony:
+		 * <ol>
+		 *   <li>
+		 *      one MUST NOT call the emit and tryEmit APIs on the sink
+		 *      <ul>
+		 *          <li>if the sink itself is unsafe (not serialized) this would result in undefined behavior</li>
+		 *          <li>if the sink is serialized, it might be ok but the source will trigger
+		 *          {@link Sinks.Many#emitNext(Object, EmitFailureHandler) calls to the emit API}, which might terminate
+		 *          the sink in case of concurrent access</li>
+		 *      </ul>
+		 *   </li>
+		 *   <li>one MUST NOT use this to connect a sink multiple times (new connections will cancel the previous subscription)</li>
+		 *   <li>this represents a virtual processor, the downstream of which is the sink's  {@link Many#asFlux()}</li>
+		 *   <li>once {@link Disposable#dispose()} is called, the sink can be reused normally</li>
```

Comment:
"reused normally" sounds a bit too reassuring to me. Wouldn't one need to know what signals the source has emitted so far in order to continue using the sink? At best it is hoping it is still possible to emit but one really doesn't know and therefore `tryEmit` should be suggested at a minimum. 

Or perhaps the wording can be adjusted further to suggest that after disconnecting it is technically okay to tryEmit again but really the main use case is to just simplify connecting to a source.